Formula/python@3.9.rb

```diff
@@ -25,7 +25,11 @@ class PythonAT39 < Formula
         You can install them, if desired, with:
           xcode-select --install
     EOS
-    satisfy { MacOS::CLT.installed? }
+    satisfy do
+      installed = on_macos { MacOS::CLT.installed? }
```

Comment:
Can we either wrap this whole thing in blocks for readability, or make this return true for Linux in brew itself.